algebird-core/src/main/scala/com/twitter/algebird/BloomFilter.scala

```diff
@@ -44,10 +44,10 @@ class RichCBitSet(val cb: CBitSet) {
 
 object BloomFilter {
 
-  def apply(numEntries: Int, fpProb: Double) = {
+  def apply[A](numEntries: Int, fpProb: Double)(implicit toBytes: A => Array[Byte]) = {
```

Comment:
Thank you for the feedback.

I'm not sure that I follow completely. Do you mean changing the signature to: `def apply[Hash128[A]](numEntries: Int, fpProb: Double)`?

And I guess that by number 2 you mean making those changes to the definitions in `algebird/algebird-core/src/main/scala/com/twitter/algebird/Hash128.scala`?

 If I understood correctly I can look into if I can get it working.